Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
+
+public class LogStore : MonoBehaviour
+{
+
+    private SortedDictionary<string, List<string>> logs;
+    private StringBuilder logString;
+    public string Label { get; set; }
+
+    private LogType logType;
+    private bool logCommonColumns;
+
+    private int nbLines;
+    private bool createStringOverTime;
+    private StringBuilder currentLineLogged;
+    private SortedDictionary<string, string> currentLogs;
+    private const string fieldSeparator = ";";
+    private const string lineSeperator = "\n";
```

Comment:
should be `lineSeparator`. (replace e with a)

Reply:
Oops 😅